pkg/discovery/worker/compliance/taint_toleration_processor.go

```diff
@@ -30,41 +31,36 @@ type TaintTolerationProcessor struct {
 	// Map of node uid indexed by node name
 	nodeNameToUID map[string]string
 
+	cluster *repository.ClusterSummary
+
 	// Manager for schedulable nodes
-	nodesManager *SchedulableNodeManager
+	unSchedulableNodesManager *SchedulableNodeManager
```

Comment:
This naming is totally off, (don't you think.. :)) with confusing readability.
Probably changing the name of the original structure to `nodesSchedulabilityManager` might solve this conflict as its dealing with both schedulable and unschedulable nodes. And then use the name `nodeManager` for the variable here.